readme/manual-setup.md

```diff
@@ -61,7 +61,7 @@ If you have already set up Ozone before you may need to clean up your local envi
 ### Step 5. Download and extract the distribution
 
 ```bash
-export VERSION=1.0.0-alpha.7&& \
+export VERSION=1.0.0-alpha.8&& \
```

Comment:
Let also change [the following lines](https://github.com/ozone-his/ozone-docker/pull/62/commits/bec1f0a32e5754ee4a1549f18514a14d77080f34#diff-dc9b156ef5fb99718ddb42433a69f750e7d4211aeaaa5b1530a2719543094dafR91-R92)

```
export O3_FRONTEND_TAG=3.0.0-beta.13;  \
export O3_BACKEND_TAG=3.0.0-beta.13;
```
to

```
export O3_FRONTEND_TAG=3.0.0-beta.16;  \
export O3_BACKEND_TAG=3.0.0-beta.16;
```